docs/releases/changelog.md

```diff
@@ -27,19 +46,125 @@ supported with NVCC + MSVC. It also adds documentation.
+    `_LIBCUDACXX_CUDA_API_VERSION_MAJOR`, `_LIBCUDACXX_CUDA_API_VERSION_MINOR`,
+    `_LIBCUDACXX_CUDA_API_VERSION_PATCH`.
+- ABI version switching: users can define `_LIBCUDACXX_CUDA_ABI_VERSION`
+    to request a particular supported ABI version.
+  `_LIBCUDACXX_CUDA_ABI_VERSION_LATEST` is set to the latest ABI version, which
+    is always the default.
+
+### Other Enhancements
 
-TODO
+- `<cuda/latch>`/`<cuda/semaphore>`: `<cuda/*>` headers added for `cuda::latch`,
+    `cuda::counting_semaphore`, and `cuda::binary_semaphore`.
+   These features were available in prior releases, but you had to include
+      `<cuda/std/latch>` and `<cuda/std/semaphore>` to access them.
+- NVCC + GCC 10 support.
+- NVCC + Clang 10 support.
 
-## libcu++ 1.1.0 (CUDA Toolkit 11.0 Early Access)
+## libcu++ 1.1.0 (CUDA Toolkit 11.0)
 
-TODO
+libcu++ 1.1.0 introduces the world's first implementation of the
+  [Standard C++20 synchronization library](https://wg21.link/P1135):
+  `<cuda/[std/]barrier>`, `<cuda/std/latch>`, `<cuda/std/semaphore>`,
+  `cuda::[std::]atomic_flag::test`, `cuda::[std::]atomic::wait`, and
+  `cuda::[std::]atomic::notify*`.
+An extension for managing asynchronous local copies, `cuda::memcpy_async` is
+  introduced as well.
+It also adds `<cuda/std/chrono>`, `<cuda/std/ratio>`, and most of
+  `<cuda/std/functional>`.
+
+### ABI Breaking Changes
+
+- ABI version 2 has been introduced and is now the default.
+  A new ABI version was introduced because it is our policy to do so in every
+    major CUDA toolkit release.
+  ABI version 1 is no longer supported.
+
+### Breaking Changes
+
+- Atomics on Pascal + Windows are disabled because the platform does not support
+    them and on this platform the CUDA driver rejects binaries containing these
+    operations.
+
+### New Features
+
+- `<cuda/[std/]barrier>`: C++20's `cuda::[std::]barrier`, an asynchronous thread
+    coordination mechanism whose lifetime consists of a sequence of barrier
+    phases, where each phase allows at most an expected number of threads to
+    block until the expected number of threads arrive at the barrier.
+  It is backported to C++11.
+  The `cuda::barrier` variant takes an additional `cuda::thread_scope` parameter.
+- `<cuda/barrier>`: `cuda::memcpy_async`, asynchronous local copies.
+  This facility is NOT for transferring data between threads or transferring
+    data between host and device; it is not a `cudaMemcpyAsync` replacement or
+    abstraction.
+  It uses `cuda::[std::]barrier`s objects to synchronize the copies.
+- `<cuda/std/functional>`: common function objects, such as `cuda::std::plus`,
+    `cuda::std::minus`, etc.
+  `cuda::std::function`, `cuda::std::bind`, `cuda::std::hash`, and
+    `cuda::std::reference_wrapper` are omitted.
+
+### Other Enhancements
+
+- C++17 support.
+- NVCC + GCC 9 support.
+- NVCC + Clang 9 support.
+- Build with warnings-as-errors.
+
+### Issues Fixed
+
+- TODO
```

Comment:
@griwes please review the changelog (`git log 1.0.0..1.1.0`) and fill this information in. You can suggest changes to the PR through GitHub's web interface.